webapp/src/containers/Rhs/projectList/index.tsx

```diff
@@ -107,7 +107,7 @@ const ProjectList = () => {
             }
             {
                 usePlugin.getApiState(plugin_constants.pluginApiServiceConfigs.getAllLinkedProjectsList.apiServiceName).isSuccess && (
-                    data && data.length > 0 ?
+                    data && data?.length > 0 ? (
                         <>
                             {
                                 data?.map((item: ProjectDetails) => (
```

Comment:
```suggestion
                                data.map((item: ProjectDetails) => (
```

Reply:
this is needed here as removing this gives `Object is possibly 'undefined'` it's type has `const data: void | UserDetails | ProjectDetails[] | undefined`